components/autofill/src/db/schema.rs

```diff
@@ -203,21 +224,61 @@ mod tests {
     }
 
     impl CreditCardRowV1 {
-        fn from_row(row: &Row<'_>) -> rusqlite::Result<CreditCardRowV1> {
-            Ok(Self {
-                guid: row.get("guid")?,
-                cc_name: row.get("cc_name")?,
-                cc_number_enc: row.get("cc_number_enc")?,
-                cc_number_last_4: row.get("cc_number_last_4")?,
-                cc_exp_month: row.get("cc_exp_month")?,
-                cc_exp_year: row.get("cc_exp_year")?,
-                cc_type: row.get("cc_type")?,
-                time_created: row.get("time_created")?,
-                time_last_used: row.get("time_last_used")?,
-                time_last_modified: row.get("time_last_modified")?,
-                times_used: row.get("times_used")?,
-                sync_change_counter: row.get("sync_change_counter")?,
-            })
+        fn from_row(row: &Row<'_>) -> Self {
```

Comment:
there's a fair bit of boilerplate there that seems like it's not going to scale that well once we get to V7. If we are leaning into the .sql files with old versions of the schemas, maybe we should also have that same .sql setup the test data? Eg, in the `_v0` file, just below the 'create table' have a few 'insert' clauses?